README.md

```diff
@@ -43,6 +43,143 @@ The documentation should help configure the BinderHub service to:
+
+   In the repository creation page, give it a name (ideally same name you are using for
+   helm chart), select 'Docker' as the format, 'Standard' as the mode, 'Region'
+   as the location type and select the same region your kubernetes cluster is in. The
+   settings about encryption and other options can be left in their default. Hit "Create".
+
+5. Find the full path of the repository you just created, by opening it in the list
+   and looking for the small 'copy' icon next to the name of the repository. If you
+   hit it, it should copy something like `<region>-docker.pkg.dev/<project-name>/<repository-name>`.
+   Save this.
+
+6. Create a Google Cloud Service Account that has permissions to push to this
+   repository ([via this URL]
+   (https://console.cloud.google.com/iam-admin/serviceaccounts/create) - make
+   sure you are in the correct project again). You may also need appropriate permissions to set this up. Give it a name (same as the name you used
+   for the helm chart, but with a '-pusher' suffix) and click 'Create and Continue'.
+   In the next step, select 'Artifact Registry Writer' as a role. Click "Next". In the final step, just click "Done".
+
+7. Now that the service account is created, find it in the list and open it. You will
+   find a tab named 'Keys' once the informational display opens - select that. Click
+   'Add Key' -> 'Create New Key'. In the dialog box that pops up, select 'JSON' as the
+   key type and click 'Create'. This should download a key file. **Keep this file safe**!
+
+8. Now that we have the appropriate permissions, let's set up our configuration! Create a
+   new file named `binderhub-service-config.yaml` with the following contents:
+
+   ```yaml
+   config:
+     BinderHub:
+       use_registry: true
+       image_prefix: <repository-path>/binder
+       # Temporarily enable the binderhub UI so we can test image building and pushing
+       enable_api_only_mode: false
+   buildPodsRegistryCredentials:
+     server: "https://<region>-docker.pkg.dev"
+     username: "_json_key"
+     password: |
+       <json-key-from-service-account>
+   ```
+
+   where:
+
+   1. `<repository-path>` is what you copied from step 5.
+
+   2. `<json-key-from-service-account>` is the JSON file you downloaded in step 7.
+      This is a multi-line file - either indent it correctly to match up (the `|`
+      allows multiline strings),
+      or simply edit the contents to be a single line. Since it is JSON,
+      it does not matter.
+
+   3. `<region>` is the region your artifact repository was created in. You can see
+      this in the first part of `<repository-path>` as well.
+
+9. Run a `helm upgrade` to use the new configuration you just created:
+
+   ```bash
+   helm upgrade \
+    --install \
+    --create-namespace \
+    --devel \
+    --wait \
+    --namespace <namespace>
+    <name> \
+    binderhub-service/binderhub-service \
+    -f binderhub-service-config.yaml
```

Comment:
I figure `--values` is better than `-f` since it helps the user understand it better.

```suggestion
    --values binderhub-service-config.yaml
```